<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="4" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="ABCD" duration-ms="188" started-at="2019-05-25T17:34:32Z" finished-at="2019-05-25T17:34:32Z">
    <groups>
    </groups>
    <test name="Reg1" duration-ms="188" started-at="2019-05-25T17:34:32Z" finished-at="2019-05-25T17:34:32Z">
      <class name="abc.testng.A4">
        <test-method status="SKIP" signature="fourTest()[pri:0, instance:abc.testng.A4@23223dd8]" name="fourTest" duration-ms="16" started-at="2019-05-25T23:04:32Z" finished-at="2019-05-25T23:04:32Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipped Test Case]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipped Test Case
	at abc.testng.A4.fourTest(A4.java:14)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:663)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:849)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1157)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1200)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1125)
	at org.testng.TestNG.run(TestNG.java:1033)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fourTest -->
      </class> <!-- abc.testng.A4 -->
      <class name="abc.testng.A3">
        <test-method status="PASS" signature="threeTest()[pri:0, instance:abc.testng.A3@16c0663d]" name="threeTest" duration-ms="0" started-at="2019-05-25T23:04:32Z" finished-at="2019-05-25T23:04:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- threeTest -->
      </class> <!-- abc.testng.A3 -->
      <class name="abc.testng.A2">
        <test-method status="PASS" signature="twoTest()[pri:0, instance:abc.testng.A2@d7b1517]" name="twoTest" duration-ms="16" started-at="2019-05-25T23:04:32Z" finished-at="2019-05-25T23:04:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- twoTest -->
      </class> <!-- abc.testng.A2 -->
      <class name="abc.testng.A1">
        <test-method status="PASS" signature="oneTest()[pri:0, instance:abc.testng.A1@2eee9593]" name="oneTest" duration-ms="47" started-at="2019-05-25T23:04:32Z" finished-at="2019-05-25T23:04:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- oneTest -->
      </class> <!-- abc.testng.A1 -->
    </test> <!-- Reg1 -->
  </suite> <!-- ABCD -->
</testng-results>
